dev/releases/create_stable_branch.py

```diff
@@ -14,81 +14,107 @@
 # TODO: implement parts of the steps described in
 # <https://github.com/gap-system/gap-distribution/blob/master/DistributionUpdate/STABLE_BRANCH_CHECKLIST.md>
 
-from utils import *
+from utils import error, notice, warning, patchfile
```

Comment:
```suggestion
from utils import error, notice, patchfile
```
`warning` isn't used